### PR TITLE
WIP: [QC-40] Add init-config command

### PR DESCRIPTION
**Pull request by calebevans**
- Add `qe-metrics init-config` command
- Add jinja2 template for config

### CODE REVIEW

File: qe_metrics/commands/init_config/__init__.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from simple_logger.logger import get_logger
+
+
+@click.argument(
+    "filepath",
+    type=click.Path(exists=False),
+    default="./qe-metrics.config",
+)
+@click.command("init-config")
+def init_config(filepath: str) -> None:
```
Comment (rnetser) [inline qe_metrics/commands/init_config/__init__.py:33]: 1. there should be one entry point
2. why is this needed? a user needs to provide the info anyhow; create an example file `qe-metrics.config.example` with the required structure. A user just needs to fill it in

**Review by rnetser — CHANGES_REQUESTED** [review]

File: qe_metrics/commands/init_config/__init__.py (hunk 1 above)
Comment (calebevans) [reply qe_metrics/commands/init_config/__init__.py:33]: 1. This or hasn't been refactored to do that, I did it in https://github.com/RedHatQE/qe-metrics/pull/4 . Thats why this one is in WIP

2. Fine by me
